File: baseball-app/src/app/contracts/remote-original.ts
import { FileType } from "./file-type";
import { RemoteFileDetail } from "./remote-file-detail";

export interface RemoteOriginal {

    fileType: FileType;
    gameName?: string;
    photo: RemoteFileDetail;
    video: RemoteFileDetail;
    alternatePhoto?: RemoteFileDetail;
    alternateVideo?: RemoteFileDetail;
}

File: BaseballApi/Api/Contracts/RemoteOriginal.cs
using System;

namespace BaseballApi.Contracts;

public struct RemoteOriginal
{
    public required string FileType { get; set; }
    public string? GameName { get; set; }
    public DateOnly? GameDate { get; set; }
    public RemoteFileDetail? Photo { get; set; }
    public RemoteFileDetail? Video { get; set; }
    public RemoteFileDetail? AlternatePhoto { get; set; }
    public RemoteFileDetail? AlternateVideo { get; set; }
}
